zarf.yaml

```diff
@@ -33,6 +33,10 @@ variables:
 
   - name: ADMIN_DOMAIN
     description: "Domain for admin services, defaults to `admin.DOMAIN`"
+  
+  - name: AIRGAP_DEPLOY
+    description: "Deploy the bundle in airgap mode. NOT INTENDED FOR PRODUCTION"
+    default: "false"
```

Comment:
if that did solve the arch thing it would be nice to use flavors instead

Reply:
otherwise we will want to test that the packages are still fast when not in airgap mode and likely split it into 2 packages.

Reply:
This actually is just here as a final method to prevent accidental airgap deploys. So you have to have the airgap flavor AND conscientiously set the AIRGAP_DEPLOY var to enable this. Might just be overly cautious with this though